### DISCUSSION TITLE
是否可以添加脚本手动更新

### DISCUSSION THREAD

**Discussion by iliaoke** [scriptscat/scriptcat#316] [General]
.

**Comment by CodFrm** [comment]
![QQ_1730359719278](https://github.com/user-attachments/assets/5ceaf987-60c7-4973-93cd-459a2f8693b8)

点击这一列，可以检查更新，多选也可以

![QQ_1730359746096](https://github.com/user-attachments/assets/5427d88e-8a0d-4b77-9130-27a14784d77b)

**Comment by iliaoke** [comment]
👌,原来在这里.如果把它摆在脚本的右上方，一键检查方便一点



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月31日(周四) 下午3:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [scriptscat/scriptcat] 是否可以添加脚本手动更新 (Discussion #316)




 
QQ_1730359719278.png (view on web)
 
点击这一列，可以检查更新，多选也可以
 
QQ_1730359746096.png (view on web)

**Reply by CodFrm** [reply]
右上方是指的哪里？

**Comment by iliaoke** [comment]
这个地方可以添加一个更新图标



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月31日(周四) 下午3:53
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [scriptscat/scriptcat] 是否可以添加脚本手动更新 (Discussion #316)




 
右上方是指的哪里？

**Reply by CodFrm** [reply]
没图，不清楚指的什么位置 😂

**Reply by iliaoke** [reply]
> 没图，不清楚指的什么位置 😂
这里。原来qq回复加附件不能同步到github

![Screenshot_2024-10-31-15-54-51-822_com yujian ResideMenuDemo-edit](https://github.com/user-attachments/assets/f908a7cc-187e-4763-86e8-32b920119c9a)

**Reply by CodFrm** [reply]
这个位置也不是很好，后续在考虑吧
